dbusapi/tests/test_interfaceparser.py

```diff
@@ -447,6 +447,41 @@ def test_doc_comments(self):
         arg = meth.arguments[0]
         self.assertEquals(arg.comment, "And me!")
 
+    def test_linenos(self):
+        """Test that line numbers are correctly computed"""
+        xml = ("<node xmlns:tp='"
+               "http://telepathy.freedesktop.org/wiki/DbusSpec#extensions-v0"
+               "' xmlns:doc='"
+               "http://www.freedesktop.org/dbus/1.0/doc.dtd'>\n"
+               "<!--\n"
+               "Please consider me\n"
+               "-->\n"
+               "<interface name='I.I'>\n"
+               "<!--\n"
+               "Notice me too\n"
+               "-->\n"
+               "<method name='foo'>\n"
+               "<!--\n"
+               "And me!\n"
+               "-->\n"
+               "<arg name='bar' type='s'/>\n"
```

Comment:
Added another argument here with *no* comment to test the default value of `comment_line_number` is correctly `-1`.